### PR TITLE
Ports "Makes felinids die from chocolate."

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Doing what @Sishen1542 should have done months prior.
Porting tgstation/tgstation#46238, also #46339, #46366 and #46790, by GuyonBroadway.

## Why It's Good For The Game
I regret nothing.

## Changelog
:cl: GuyonBroadway
balance: Felinids now have a very adverse reaction to chocolate products.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/felinid.dm
Comment (Trilbyspaceclone) [inline code/modules/mob/living/carbon/human/species_types/felinid.dm]: Forgot villina

**Comment by Thalpy** [comment]
As this is a nerf to felinids, perhaps you could propose a counter buff, to keep things balanced? Purrhaps a purr action button?

File: code/modules/mob/living/carbon/human/species_types/felinid.dm
Comment (Ghommie) [reply code/modules/mob/living/carbon/human/species_types/felinid.dm]: Vanilla is not chocolate toh.

**Comment by PersianXerxes** [comment]
Bad meme >:(

**Comment by YakumoChen** [comment]
Can we not

**Comment by Tename** [comment]
Don't forget dogs too.

**Comment by Ghommie** [comment]
> 
> 
> Don't forget dogs too.

I mean, we could make all antropomorphs die from chocolate too. That'd only upset 90% of the community and not just felinid mains. :smiling_imp: 

> 
> 
> As this is a nerf to felinids, perhaps you could propose a counter buff, to keep things balanced? Purrhaps a purr action button?

Countering gameplay drawbacks with roleplay buffs? Really?

**Comment by Thalpy** [comment]
> Countering gameplay drawbacks with roleplay buffs? Really?

Clearly the purr would allow for some mechanical self heal, or the ability to reflect bullets from the internal self resonance causing projectiles to hit their resonance frequency causing their inertia to invert.

I was being silly Ghommie, though I think it's due diligence to buff the poor kitties if this is taken seriously. It also feels pretty mean to displace cats from valentines day, have you thought of adding cat safe chocolates too?

**Comment by xurxomario** [comment]
Wasnt this a "felinids bad" salt pr back there?

But hey, regardless of that, if we are gonna meme felinids may as well go the whole nine yards, give em the ability to spit out non-damaging furballs or something similar for the giggles.

**Comment by Ghommie** [comment]
> 
> 
> Wasnt this a "felinids bad" salt pr back there?

Nah, new players have been restricted from making felinid characters on tgstation for more than a year now. No real salt, just for fun.

**Comment by ShadeAware** [comment]
It is now time for execution by chocolate.

**Comment by lolman360** [comment]
at worst people are going to use chocolate as a unique and fun poison, what's wrong with that?

**Comment by Ghommie** [comment]
You can always make a revert as soon as this PR is merged (implying it even will).

**Review by silicons — CHANGES_REQUESTED** [review]
dude just make it do slight tox and maybe a message/stunless throwups none of the ridiculous organ damage/disgust mess jesus dude

**Comment by lolman360** [comment]
>balance that directly affects kevinz
>kevinz: "reeeee"

lmao

**Comment by silicons** [comment]
dude that's still organ damage which is a horrible idea with where our medical code is is there a reason to not do tox damage?

**Comment by Ghommie** [comment]
Low toxin damage is easy to counter, nausea slowly leads to emesis and purge of the mob's reagents, also charcoal.
A normal felinid mob has 10 internal organs, excluding genitalia, a complexive pool of 1000 or so hitpoints against which 5 damage per tick is fairly low.
While the effects are stackable with multiple chocolate reagents. The disgust will inevitably end up clearing them at one point or another.

And we both know that this will be reverted by someone else if ever merged, no need to take this seriously. :clown_face:

**Comment by Poojawa** [comment]
This is a shit meme, because if we're gunna try to finger point chocolate being toxic to cats/dogs, you have to realize it's about dosage and bodyweight for theobromine poisoning.

if a saint bernard eats a fuckin' milk dud, it isn't going to drop dead.

This is just shitty tg 'REEEE FURRIES' bullshit.

**Comment by optimumtact** [comment]
it shouldn't have been merged in the first place

**Comment by Ghommie** [comment]
We all let crap fly past the radar once in a while.
Maintainers go do whatever.

**Comment by Ghommie** [comment]
This meme is not a thing anymore on tgstation, took ne a while to notice.
Hoke's old, the needless debate also made it even more boring.

**Pull request closed without merging**
